source/reference/api/start.txt

```diff
@@ -100,6 +100,15 @@ Request Body Parameters
        
        Default value is ``false``.
 
+   * - ``sharding``
+     - document
+     - Optional
+     - Configure synchronization between a replica set and sharded cluster.
+       Synchronization from a replica set to a sharded cluster requires this
+       option.
+
+       See :ref:`c2c-api-start-sharding` for more information.
```

Comment:
[suggestion, non-blocking]

Our [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear) states that links should begin with the purpose of the link, rather than the link itself.

Suggest swapping the order here to:

For more information, see :ref:`c2c-api-start-sharding`.

Reply:
Fixed